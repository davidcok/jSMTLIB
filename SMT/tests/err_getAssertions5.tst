; interactive-mode false
(set-option :interactive-mode false)
(set-logic QF_UF)
(get-assertions)
