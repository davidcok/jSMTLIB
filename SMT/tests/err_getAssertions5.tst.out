success
success
(get-assertions)
^^^^^^^^^^^^^^^^
(error "The get-assertions command is only valid if :interactive-mode has been enabled")
